var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -59,8 +61,12 @@ class Sirius(CMakePackage, CudaPackage):
     depends_on('py-pybind11', when='+python', type=('build', 'run'))
     depends_on('magma', when='+magma')
 
-    depends_on('spfft', when='@6.4.0:~cuda')
+    depends_on('spfft', when='@6.4.0:')
+    depends_on('spfft', when='@develop')
+    depends_on('spfft', when='@master')
```

Comment:
Both `master` and `develop` are included in `@6.4.0:`, so you don't need these extra lines.